src/main/java/cat/udl/eps/softarch/demo/domain/Client.java

```diff
@@ -1,21 +1,30 @@
 package cat.udl.eps.softarch.demo.domain;
 
-import jakarta.persistence.ElementCollection;
-import jakarta.persistence.Entity;
+import jakarta.persistence.*;
```

Comment:
You should import the minimum dependencies in your project. For that, use the shortcut ⌃⌥O (macOS) / Ctrl+Alt+O.